CONTRIBUTORS.txt

```diff
@@ -154,6 +154,9 @@ In chronological order:
 * Jon Wayne Parrott <jonwayne@google.com>
   * App Engine environment tests.
 
+* James Atherfold <jlatherfold@hotmail.com>
+  * Bugfixes.
```

Comment:
Ok done. Feel free to change it if you can come up with something that fits better.